plugins/workflow/featuring/featuring.php

```diff
@@ -309,11 +309,11 @@ public function onWorkflowBeforeTransition(WorkflowTransitionEvent $event)
 	 *
 	 * @param WorkflowTransitionEvent $event
 	 *
-	 * @return boolean
+	 * @return   void
```

Comment:
```suggestion
	 * @return  void
```
This is to align the 2nd column.

Reply:
Well the `@param` is in violation of the code standards, and what you are suggesting is also in violation of them too. This PR passes the code standards and there is already a different PR to make the `@param` valid with the code standards of Joomla. https://github.com/joomla/joomla-cms/pull/30091/files#diff-c87bf8acf02b4fce312827b3d9dd7448R310